.github/workflows/ci.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.8', '3.11']
-        toxenv: [quality, docs, pii_check, django42]
+        python-version: ['3.10', '3.11']
+        toxenv: [quality, docs, pii_check, django52]
```

Comment:
We should add support for testing with django52 without removing django42 support for now, unless there is a backwards incompatible change that has to be made.